provisioner/data.js

```diff
@@ -211,39 +211,31 @@ WorkerType.create = function (workerType, properties) {
 /**
  * Return a list of all known workerTypes
  */
-WorkerType.loadAll = function () {
-  var workers = [];
+WorkerType.loadAll = async function () {
+  let workers = [];
 
-  var p = base.Entity.scan.call(this, {}, {
+  await base.Entity.scan.call(this, {}, {
     handler: function (item) {
       workers.push(item);
     },
```

Comment:
You'll enjoy `{handler(item) { workers.push(item); }}` or `{handler: item => workers.push(item)}` even more :)

First one uses method in dictionary syntax...